src/api/constants.ts

```diff
@@ -1,7 +1,28 @@
-export const CLIENT_ID = 'JpwJuEhdlzYrH6FO3kW6QVP8GTr0cVr9';
-export const CLIENT_SECRET =
-  'rctURI8w_fRJe73DPqaI8ASCAbJGCd18p8QsU9nkNpU1_CTadZUzsWzD79YtovLO';
-export const AUDIENCE = 'https://dev-y4thf51u18ja0ber.eu.auth0.com/api/v2/';
-export const SCOPE = 'openid offline_access';
+import { Configuration, ProtocolMode } from '@azure/msal-browser';
+import {
+  REACT_APP_CLIENT_ID,
+  REACT_APP_AUTHORITY,
+  BASE_APP_URL,
+} from '../windows';
+
+export const SCOPE = ['openid', 'offline_access', 'email'];
```

Comment:
Here, you have the option to provide multiple scopes. So, the variable must be renamed accordingly.